hasher-matcher-actioner/hmalib/common/config.py

```diff
@@ -137,6 +137,14 @@ def get_all(cls: t.Type[TConfig]) -> t.List[TConfig]:
                     ret.append(obj)
         return ret
 
+    @classmethod
+    def exists(cls: t.Type[TConfig], name: str) -> bool:
+        _assert_initialized()
+        ret = cls.get(name)
+        if not ret:
+            return False
+        return True
```

Comment:
For whatever reason I was afraid to try `return not not ret`. Is that something I should try?

Reply:
worth double checking based on the impl of get but I think `return bool(cls.get(name))` would work

Reply:
Ok. Using `return bool(cls.get(name))`